src/test/test-utils.ts

```diff
@@ -113,7 +113,7 @@ export class InMemoryCruxUserRepository implements ICruxUserRepository {
             }
         },{
             configuration: {
-                blacklistedCruxUsers: [],
+                // blacklistedCruxUsers: [],
```

Comment:
Don't comment anything, just remove your changes. Let it be the way it is. You have nothing to do with this file.

Reply:
done